plugins/source/unit_test_validators.go

```diff
@@ -0,0 +1,98 @@
+package source
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/cloudquery/plugin-sdk/schema"
+)
+
+type Validator func(t *testing.T, plugin *Plugin, resources []*schema.Resource) error
+
+func validateColumnsHaveData(t *testing.T, plugin *Plugin, resources []*schema.Resource) error {
+	t.Helper()
+	tables := extractTables(plugin.tables)
+	for _, table := range tables {
+		err := validateTable(t, table, resources)
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func validateTableNamePlural(t *testing.T, plugin *Plugin, resources []*schema.Resource) error {
+	t.Helper()
+	tables := extractTables(plugin.tables)
+	for _, table := range tables {
+		if !strings.HasSuffix(table.Name, "s") {
```

Comment:
Should we use https://github.com/gertd/go-pluralize (or any other similar lib) to check this?
Otherwise we're only testing that tables end with `s`

Reply:
We can definitely do something more substantial, I was just pulling the same functionality that we had in codegen here...